tools/api-builder/angular.io-package/tag-defs/index.js

```diff
@@ -1,6 +1,6 @@
 module.exports = [
   require('./deprecated'),
   require('./howToUse'),
-  require('./whatItIs'),
+  require('./whatItDoes'),
```

Comment:
I realized the naming mistake after looking more closely at the api doc proposal document in https://github.com/angular/angular.io/issues/964